guides/transfer_learning.py

```diff
@@ -80,7 +80,7 @@
 non-trainable weights is the `BatchNormalization` layer. It uses non-trainable weights
  to keep track of the mean and variance of its inputs during training.
 To learn how to use non-trainable weights in your own custom layers, see the
-[guide to writing new layers from scratch](making_new_layers_and_models_via_subclassing).
+[guide to writing new layers from scratch](custom_layers_and_models).
```

Comment:
The link here should be to https://keras.io/guides/making_new_layers_and_models_via_subclassing/

Please replace it with the full link.

Reply:
Thanks @fchollet , I have updated it.